cognite_toolkit/_cdf_tk/loaders/_base_loaders.py

```diff
@@ -328,6 +328,10 @@ def load_resource(self, resource: dict[str, Any], is_dry_run: bool = False) -> T
         """Loads the resource from a dictionary. Can be overwritten in subclasses."""
         return self.resource_write_cls._load(resource)
 
+    def load_minimum_resource(self, resource: dict[str, Any]) -> T_ID:
```

Comment:
I don't think this is necessary. The `self.get_id` supports dict` directly. 
So you can do 
```python
my_id = loader.get_id({"externalId": "some-id"})
```